### ISSUE TITLE
add emoji code for flag of sri lanka

### ISSUE THREAD

**Issue by afazdeen** [99xt/emojicpp#22]

**Comment by anjulalk** [comment]
Does this issue refer to the emojis shown in README file?
@lakindu95

**Comment by isurunuwanthilaka** [comment]
Code already added in the emoji.h header file
`{":flag_for_Sri_Lanka:" , "\U0001F1F1\U0001F1F0"}`

**Comment by shalithasuranga** [comment]
@isurunuwanthilaka  yeah  header file already got correct unicodes for Sri Lanka flag. Btw we can use this issue to add it to README as @anjulalk  mentioned 
![image](https://user-images.githubusercontent.com/18402702/85283911-d0942b00-b4ab-11ea-8233-361b835cf00e.png)

**Comment by ilka-schulz** [comment]
> Code already added in the emoji.h header file `{":flag_for_Sri_Lanka:" , "\U0001F1F1\U0001F1F0"}` @shalithasuranga

Tbh, I can not find it in the `emoji.h` at the tip of the current default branch (28954edebbcb34a4e1957ccc07e02a8423e1aa1b). Has this been undone at some point in time?

**Comment by shalithasuranga** [comment]
Closing this since the emoji is added.. Thanks all
